### DISCUSSION TITLE
Do we need a corporate "sponsor"?

### DISCUSSION THREAD

**Discussion by becarpenter** [becarpenter/book6#3] [Ideas]
Would it be advantageous to have this effort sponsored by, for example, the Internet Society? This could give us advantages such as reputation, modest funding, maybe web hosting, and legal advice.
Alternatively, would it be advantageous to have a relationship with a tech publisher like Manning or O'Reilly? Would that work with an open source approach?

**Comment by xipengxiao** [comment]
Sponsorship from ISOC may be good.  No idea whether publisher will work with such an open source approach

**Comment by becarpenter** [comment]
A regular publisher might indeed have a problem with open source. There are "print on demand" publishers (the modern version of a so-called "vanity publisher") who probably don't care so much. I will ask somebody with a bit more publishing experience.

**Reply by becarpenter** [reply]
Based on initial feedback from ISOC people, there's a good chance of collaboration there, but it's summer vacation time so it may take a bit longer to reach a conclusion.

**Comment by buraglio** [comment]
My strong preference is toward an open source approach, can be published as an epub or the like pretty easily. Amazon has a print on demand option as well.

**Reply by becarpenter** [reply]
No dispute about that in principle. There are some sordid details like how to generate the printable format and how it can be always up to date. Nothing worse than printing something that's already out of date.

**Comment by buraglio** [comment]
I know a few folks that have used leanpub, which auto-updates, but I have
not used it personally.

**Comment by cgaylord-vt** [comment]
So on this point, I posed the question to Brian specifically to the question of whether the marketing oomph of a Manning or O'Reilly would be valuable. I am also more religiously aligned with an open source approach, but as the intent is to get this into the hands of the IT Professional, I'm sympathetic to the point that the involvement of a true publication house might have value.

**Reply by becarpenter** [reply]
The question is whether a for-profit publisher will accept the idea of an open-source book with continuous updates. I honestly don't know. I think the only way to find out is to flesh out the idea a bit more (thanks, Nick, for your work so far) and then approach a couple of publishers to see.

**Comment by becarpenter** [comment]
FYI, for the moment we are continuing without a sponsor. And not worrying about it. However, it doesn't seem that free hosting on GitHub will work for massive numbers of readers, so we should circle back to this issue when we have more text to show.

**Comment by xipengxiao** [comment]
Hi Brian,

I agree with this plan.  I am confident we can find a host for the book.  ISOC has this IPv6 page https://www.internetsociety.org/deploy360/home/operators/ipv6/ .  In the worst case, we can put it there.

XiPeng

**Comment by cgaylord-vt** [comment]
C.f. Eric Raymond's New Hackers Dictionary, which was a point in time printing of the jargon file/catb.org On the one hand I think we may be able to find a commercial publisher that is interested; on the other I think the business of publishing is something that is obscure enough that we may want to shop the concept sooner than later.

**Comment by becarpenter** [comment]
If we want to retain open access, I really doubt that commercial publishers will be interested. The "business" model of open access academic journals might be worth investigating. But we'll be in a much stronger position when we have the real makings of a book to show... so get writing :-).

**Comment by timchown** [comment]
Indeed, it gas to be both open access and we should probably allow a means to print a copy at cost.

There’s enough commercial books on IPv6; our ’selling point’ should be current and constantly updated content freely available.

Tim

**Comment by becarpenter** [comment]
I'm locking this discussion because it seems to have been hijacked in some mysterious way. Collaborators can still use it.

**Comment by becarpenter** [comment]
I conclude that we are managing fine without a sponsor. As long as GitHub survives...
